pages/advanced/minecraft-proxying.mdx

```diff
@@ -0,0 +1,23 @@
+# How to proxy a minecraft server
+
+If you are looking of how to access your minecraft server without your server port, and using your own domain, you are at the right place
+
+1. You need to have a minecraft server 
+2. If you already use cloudflare you are good 👍 If you use your own domain dash, make sure that it allows you to create srv records
+3. Create an srv record and fill the boxes with the information required:
+```
+Type: SRV
+Name: [your subdomain name]
+Service: _minecraft 
+Protocol: TCP (remember tcp is only for java edition, bedrock uses udp)
+TTL: auto
+Priority: 0
+Weight: 0
+Port: [Your server port]
+Target: [Domain name of the server] (most likely it will be dono-02.danbot.host)
```

Comment:
Game servers are also on dono 4 as well, so that can be added here as well.

Reply:
Ah, alr. I'll update that now.